layouts/partials/head.html

```diff
@@ -15,10 +15,10 @@
     <!-- Title and meta description
     ================================================== -->
     <title>{{ .Site.Title }} | {{ .Site.Params.Owner }}</title>
-    <meta property="og:title" content="{{ .Site.Params.Title }}" />
+    <meta property="og:title" content="{{ $translation.meta.title }}" />
     <meta property="og:url" content="{{ .Site.BaseURL }}" />
-    <meta name="description" content="{{ .Site.Params.Description }}" />
-    <meta property="og:description" content="{{ .Site.Params.Description }}" />
+    <meta name="description" content="{{ $translation.meta.description }}" />
+    <meta property="og:description" content="{{ $translation.meta.description }}" />
```

Comment:
This is great! Bringing the `meta.description` and `meta.title` is a great approach for consistent sharing across different languages. Spanish Facebook shares will get the correct language and vice-versa.